src/Sentry/ISentryClient.cs

```diff
@@ -52,6 +54,16 @@ public interface ISentryClient
     [EditorBrowsable(EditorBrowsableState.Never)]
     void CaptureTransaction(SentryTransaction transaction, Scope? scope, Hint? hint);
 
+    /// <summary>
+    /// Captures one or more metrics to be sent to Sentry.
+    /// </summary>
+    void CaptureMetrics(IEnumerable<Metric> metrics);
```

Comment:
could we get around these public methods by making the envelope outside (on hub) and just calling capture envelopes?

Reply:
Simple but cunning... why did I not think of that 😌